lib/publify/api.rb

```diff
@@ -0,0 +1,14 @@
+require 'net/http'
+
+module Publify
+  class API
+    def self.latest_links(limit)
+      uri = URI("http://#{PUBLIFY_HOSTNAME}/articles.json")
+      response = Net::HTTP.get uri
```

Comment:
@neoeno We're not referring to the PUBLIFY_HOSTNAME correctly which is causing the issue.

@benlovell Can you clarify what you mean by "unbounded"? Currently the call is made for each article that is rendered on frontend.